packages/ove-app-audio/src/client/constants/audio.js

```diff
@@ -1,44 +1,44 @@
 const Constants = {
     /**************************************************************
-                                Viewer
-    **************************************************************/
+     Viewer
+     **************************************************************/
     BUFFER_STATUS_BROADCAST_FREQUENCY: 700,
     RESCALE_DURING_REFRESH_TIMEOUT: 1000,
 
     /**************************************************************
-                              Controller
-    **************************************************************/
+     Controller
+     **************************************************************/
     DEFAULT_STATE_NAME: 'WikiCommonsGoingHome',
 
     /**************************************************************
-                             Audio Player
-    **************************************************************/
+     Audio Player
+     **************************************************************/
     VOLUMEUP_MULTIPLIER: 1.2,
```

Comment:
Why exactly do we increase/decrease volume using different multipliers? Like to understand the benefit of doing this compared to have one multiplier and then using division instead of multiplication in the volDown function.

Reply:
Also, ``1/VOLUMEDOWN_MULTIPLIER =  1/0.8 = 1.25 != 1.2 = VOLUMEUP_MULTIPLIER``

This means that a volumeup immediately followed by a volumedown sets the audio to ``0.8*1.2 = 0.96`` times its original level.

Reply:
@davidbirchwork, constants.js should let you do that, actually. But, there is no harm in using a single variable, since it is easier to follow the code that way.

Reply:
sure, this might just be ESLint 
![image](https://user-images.githubusercontent.com/11289564/47910715-9bc50180-de8b-11e8-9d78-76925faa275b.png)

Reply:
sadly that's not going to work either in .js. 
![image](https://user-images.githubusercontent.com/11289564/47913530-61139700-de94-11e8-9690-96d593b86dcb.png)

Reply:
have you tried "this"?

Reply:
thanks @miguems though sadly that also appears to be no good in this context 
![image](https://user-images.githubusercontent.com/11289564/47913967-d59b0580-de95-11e8-91d6-0f2629b926c1.png)

Reply:
at least now isn't red :D

Reply:
lol ;)

Reply:
either way this issue is resolved :)

Reply:
I think perhaps the way it is defined could be improved to Constants = {}, and then Constants.A = 'xyz'; Constants.B = 'abc'; etc. But, I think that's too ugly as well. I don't think there is an easy way of doing what we want to do here, which is to use a key when it is _being defined_. 

Constants anyway is not defined in a global context, it is embedded into other objects from the framework. The error is because the linters are not application-aware.

The issue is resolved anyway, so it's ok, :).